source/libs/utils.ts

```diff
@@ -207,6 +207,9 @@ export function getLatestVersionTag(tags: string[]): string {
 
 const escapeRegex = (string: string) => string.replace(/[\\^$.*+?()[\]{}|]/g, '\\$&');
 const prCommitRegex = new RegExp(`\\b${escapeRegex(location.origin)}[/][^/]+[/][^/]+[/]pull[/]\\d+[/]commits[/][0-9a-f]{7,40}\\b(?! \\]|\\))`, 'gi');
```

Comment:
Instead of having a dedicate function, you can export this:
```suggestion
export const prCommitRegex = new RegExp(`\\b${escapeRegex(location.origin)}[/][^/]+[/][^/]+[/]pull[/]\\d+[/]commits[/][0-9a-f]{7,40}\\b(?! \\]|\\))`, 'gi');
```

And then use it as `prCommitRegex.test(field.value)`